models/app.model.js

```diff
@@ -56,3 +56,12 @@ exports.checkUserExists = (username) => {
     return rows;
   })
 }
+
+exports.updateVotes = (article_id, inc_votes) => {
+  if(!inc_votes){
+    return Promise.reject({status: 400, msg: "Bad Request: inc_votes field is missing"});
+  }
```

Comment:
You could let psql deal with this - if you try to patch without this value psql will throw an error itself rather than you doing it manually!